### PR TITLE
tracee-ebpf: add --capture ignore

### PR DESCRIPTION
**Pull request by josedonizetti**
Fixes https://github.com/aquasecurity/tracee/issues/876

### CODE REVIEW

File: tracee-ebpf/tracee/tracee.go [hunk 1]
```diff
@@ -134,8 +134,13 @@ type netProbe struct {
 
 // Validate does static validation of the configuration
 func (cfg OutputConfig) Validate() error {
-	if cfg.Format != "table" && cfg.Format != "table-verbose" && cfg.Format != "json" && cfg.Format != "gob" && !strings.HasPrefix(cfg.Format, "gotemplate=") {
-		return fmt.Errorf("unrecognized output format: %s. Valid format values: 'table', 'table-verbose', 'json', 'gob' or 'gotemplate='. Use '--output help' for more info.", cfg.Format)
+	if cfg.Format != "table" &&
+		cfg.Format != "table-verbose" &&
+		cfg.Format != "json" &&
+		cfg.Format != "gob" &&
+		cfg.Format != "ignore" && // TODO: add to test after merging https://github.com/aquasecurity/tracee/pull/881
```
Comment (grantseltzer) [inline tracee-ebpf/tracee/tracee.go:141]: We try to avoid TODO's in comments, better to just make a github issue

Comment (josedonizetti) [reply tracee-ebpf/tracee/tracee.go:141]: @grantseltzer okay, but the PR was just merged, I’ll rebase and resolve the TODO. Thanks

**Review by itaysk — COMMENTED** [review]
Thanks for this PR and your other recent ones as well!

We need to consider a few things here:
1. do we want to turn off output events and errors? I think just events
2. regardless of (1), this is not an output format, so the config is in the wrong place.

If we do want to toggle just the output events, I think it belongs under the "out-file" setting. If we want to toggle everything, then it belongs at the top level of "output".

**Comment by josedonizetti** [comment]
@itaysk Flag moved to `out-file`, but it was looking a bit weird to pass `out-file:ignore`, so instead I changed it to be `out-file:/dev/null` it reads better considering the option has `file` on the name. Also, ignorePrinter is only ignoring events, not errors, but if in the future it becomes a need to also ignore errors, we could use the same syntax `err-file:/dev/null`.

**Comment by itaysk** [comment]
I'm not sure about that.. using a real path name as a special option that changes the behavior does not sound best to me (others please chime in). I'd prefer `-o out-file:none` if we are using the out-file argument.

But after thinking about this abit, I think that the `--output` flag should just control the main output - stdoud, and not the errors. If I'm consuming events over a pipe with gob, it doesn't mean I want the errors in gob as well. Errors are usually consumed by a different person, tool, etc and should also be printed to stderr as "regular" text messages. This could be done in a separate PR, but it means that in this PR we can choose to do `-o none` which will read better for you

**Comment by josedonizetti** [comment]
@itaysk about printing to stdout/stderr, there is a separate issue to fix it https://github.com/aquasecurity/tracee/issues/752, what I didn’t understand is if you prefer ‘-o none’ or ‘-o out-file:none’

**Comment by itaysk** [comment]
>I didn’t understand is if you prefer ‘-o none’ or ‘-o out-file:none’

`-o none`

**Comment by grantseltzer** [comment]
I'm going to have to agree that  `-o out-file:/dev/null` and `-o none` should be two separate things. What if someone is using tracee and wants to see the writes without saving to disk?

**Review by yanivagman — COMMENTED** [review]
LGTM
Can you please also add this option to the docs (in tracee-ebpf/output.md)?

**Review by yanivagman — APPROVED** [review]

**Pull request merged**